crates/forge/tests/integration/cheat_execution_info.rs

```diff
@@ -100,178 +100,6 @@ fn start_and_stop_cheat_transaction_hash_single_attribute() {
     assert_passed(&result);
 }
 
-#[test]
-#[allow(clippy::too_many_lines)]
-fn start_cheat_execution_info_all_attributes_mocked() {
-    let test = test_case!(
```

Comment:
Let's `#[ignore]` these removed tests instead and add a TODO comment with relevant issue linked